crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,6 +1156,19 @@ impl World {
         result
     }
 
+    /// "Sends" an `event`.
+    #[inline]
+    pub fn event_send<E: crate::event::Event>(&mut self, event: E) {
+        self.resource_mut::<crate::event::Events<E>>().send(event);
+    }
+
+    /// Sends the default value of the event.
```

Comment:
```suggestion
    /// Sends the default value of the [event](crate::event).
```

Same.

Reply:
I ended up linking directly to `Event`, which links to `Events` which has usage examples.